elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -991,6 +991,16 @@ elytron.failover-realm.delegate-realm=Reference to the security realm to use as
 elytron.failover-realm.failover-realm=Reference to the security realm to use in case the delegate-realm is unavailable.
 elytron.failover-realm.emit-events=Whether SecurityEvent signifying delegate-realm unavailability should be emitted.
 
+elytron.jaas-realm=A JAAS realm definition. It wraps a LoginContext initialized from the provided jaas-config-path.
```

Comment:
In this context, it is not clear at the first glance what jaas-config-path is. I suggest change it to something like "JAAS login configuration file".

Reply:
@OndrejKotek  This line refers to the name of the realm resource. Updated to make it clearer. Let me know if it is now OK. Thanks!